Formula/infer.rb

```diff
@@ -29,33 +35,42 @@ def install
     # Apple's libstdc++ is too old to build LLVM
     ENV.libcxx if ENV.compiler == :clang
 
-    opamroot = buildpath/"opamroot"
-    opamroot.mkpath
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["sqlite"].opt_lib/"pkgconfig"
+
+    # opamroot = HOMEBREW_CACHE/"opam"
+    opamroot = libexec/"opam"
     ENV["OPAMROOT"] = opamroot
     ENV["OPAMYES"] = "1"
 
-    # do not attempt to use the clang in facebook-clang-plugins/ as it hasn't been built yet
-    ENV["INFER_CONFIGURE_OPTS"] = "--prefix=#{prefix} --without-fcp-clang"
+    # explicitly build the clang before infer's configure
+    ENV["INFER_CONFIGURE_OPTS"] = "--prefix=#{prefix} --with-fcp-clang"
 
     llvm_args = %w[
       -DLLVM_INCLUDE_DOCS=OFF
```

Comment:
```suggestion
```

This seems to already be set: https://github.com/facebook/facebook-clang-plugins/blob/eeac101a411b4c82380cbcf326c13fa75f5b2063/clang/setup.sh#L112